src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,60 @@
-export const Tabs = () => {};
+import React, { Dispatch, SetStateAction } from 'react';
+import classNames from 'classnames';
+
+interface Tab {
+  id: string;
+  title: string;
+  content: string;
+}
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: string;
+  selectedTab: Tab | undefined;
+  onTabSelected: Dispatch<SetStateAction<string>>;
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  selectedTab,
+  onTabSelected,
+}) => {
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
+              className={
+                classNames({
+                  'is-active': tab.id === selectedTabId,
+                })
+              }
+              data-cy="Tab"
+            >
+              <a
+                href={`#${tab.id}`}
+                data-cy="TabLink"
+                onClick={
+                  () => {
+                    if (tab.id !== selectedTabId) {
+                      onTabSelected(tab.id);
+                    }
+                  }
+                }
+              >
+                {tab.title}
+              </a>
+            </li>
+          ))}
+        </ul>
+      </div>
+
+      <div className="block" data-cy="TabContent">
+        {selectedTab && selectedTab.content}
```

Comment:
```suggestion
        {selectedTab.content}
```
This will be the content of the first tab if no other tab is selected